t/server-iterate-singlestep.t

```diff
@@ -24,15 +25,30 @@ is($client->{client}->connect_async(
     $data
 ), STATUSCODE_GOOD, "connect async");
 
-while ($client->{client}->getState() != CLIENTSTATE_SESSION) {
-    $client->{client}->run_iterate(0);
+my $i;
+for ($i = 50; $i > 0; $i--) {
+    my $sc = $client->{client}->run_iterate(0);
     $server->step();
+
+    if ($client->{client}->getState() == CLIENTSTATE_SESSION) {
+    	pass "server: client session established";
+    	last;
+    }
+    note "server iteration: $i";
+    sleep(.1);
+}
+
+if ($i == 0) {
+    fail "server: loop timeout. client could not establish a session";
```

Comment:
The fail and pass output should have the same name.  Then the framework appends "ok" or "not ok".

Reply:
Fixed.